### ISSUE TITLE
[Bug Report] datetime-picker takes milliseconds in the selected time

### ISSUE THREAD

**Issue by SilentTiger** [ElemeFE/element#5616]
### Element UI version
1.3.4

### OS/Browsers version
any

### Vue version
2.2.6

### Reproduction Link
[https://jsfiddle.net/silenttiger/1axnbbdk/](https://jsfiddle.net/silenttiger/1axnbbdk/)

### Steps to reproduce
在日期时间组件中选择一个精确到秒的时间点

### What is Expected?
在日期时间组件中选择一个精确到秒的时间点之后，从组件中获取的Date对象的毫秒栏位应该是 0

### What is actually happening?
在日期时间组件中选择一个精确到秒的时间点之后，从组件中获取的Date对象的毫秒栏位不是 0
而且这个毫秒栏位的值是怎么来的还不太清楚

**Comment by ramonmata** [comment]
Millisecond field comes because the linked model "input" is empty, then picker initializes internally with the current date/time.

As a workaround, if you do initialize the input field with the date, you can customize the milliseconds

```
var Main = {
  data() {
    return {
      input: ''
    }
  },
  created() {
    var initialDate = new Date()
    initialDate.setMilliseconds(0)
    this.input = initialDate
  }
}
var Ctor = Vue.extend(Main)
new Ctor().$mount('#app')
```

**Comment by SilentTiger** [comment]
@ramonmata Thanks for your reply.
In some case, we need the default value of the datepicker is empy, and then let users select a specific datetime like "2017-06-30 09:00:00", so we can't set a default Date and make the millisecond to 0.
So, I still think this is a bug. it will make the backend server need to treat the value again to make the millisecond is 0.

**Comment by Leopoldthecoder** [comment]
Fixed in #5663.

**Comment by goatandsheep** [comment]
This is messing with my code as I need to modify the milliseconds. Instead of setting it to 0, can we make it so it's only set to 0 if the format includes it, i.e. HH:mm:ss:SSS?
